<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRtmpStreamerClass</class>
 <widget class="QWidget" name="XRtmpStreamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XRtmpStreamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
QWidget
{
	background-color: rgb(53, 53, 53);
}

QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QLineEdit
{
border-radius:10px;
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}
QComboBox
{
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>推流RTMP URL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入的URL或0，1:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜级别:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="faceLevel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三级</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="outUrl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>rtmp://192.168.174.129/live/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inUrl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XRtmpStreamer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>XRtmpStreamerClass</receiver>
   <slot>startStream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStream()</slot>
 </slots>
</ui>
